content/providers/quick-start/data.yaml

```diff
@@ -31,26 +31,26 @@
       from qiskit.primitives import Estimator
       estimator = Estimator()
 
-      # Express hydrogen molecule Hamiltonian as an operator
+      # Specify problem hamiltonian
       from qiskit.quantum_info import SparsePauliOp
-      H2_operator = SparsePauliOp.from_list([
-          ("II", -1.052373245772859),
-          ("IZ", 0.39793742484318045),
-          ("ZI", -0.39793742484318045),
-          ("ZZ", -0.01128010425623538),
-          ("XX", 0.18093119978423156)
-      ])
-
-      # Calculate ground state energy using VQE
-      from qiskit.circuit.library import TwoLocal
-      from qiskit.algorithms.optimizers import SLSQP
-      from qiskit.algorithms.minimum_eigensolvers import VQE
-
-      ansatz = TwoLocal(num_qubits=2, rotation_blocks="ry", entanglement_blocks="cz")
-      optimizer = SLSQP(maxiter=100)
-      vqe = VQE(estimator, ansatz, optimizer)
-      result = vqe.compute_minimum_eigenvalue(operator=H2_operator)
-      print(result.eigenvalue)
+      hamiltonian = SparsePauliOp.from_list(
+          [("YZ", 0.3980), ("ZI", -0.3980), ("ZZ", -0.0113), ("XX", 0.1810)]
+      )
+
+      # Define VQE ansatz, initial point and cost function
+      from qiskit.circuit.library import EfficientSU2
+      ansatz = EfficientSU2(hamiltonian.num_qubits)
+      initial_point = [0]*16
+      def cost_function(params, ansatz, hamiltonian, estimator):
+          energy = estimator.run(ansatz, hamiltonian, parameter_values=params).result().values[0]
+          return energy
+
+      # Run VQE using SciPy minimizer routine
+      from scipy.optimize import minimize
+      result = minimize(cost_function, initial_point, args=(ansatz, hamiltonian, estimator), method="cobyla")
+
+      # Print minimum eigenvalue
+      print(result.fun)
```

Comment:
Now that I see the side-by-side comparison, I am thinking it might make sense to actually keep the hamiltonian and ansatz from the old example (might be easier with those familiar with the examples). In that case, the snippet would look like this:

```suggestion
      hamiltonian = SparsePauliOp.from_list(
          ("II", -1.052373245772859),
         ("IZ", 0.39793742484318045),
         ("ZI", -0.39793742484318045),
         ("ZZ", -0.01128010425623538),
         ("XX", 0.18093119978423156)
     ])
      
      # Define VQE ansatz, initial point and cost function
      from qiskit.circuit.library import TwoLocal
      ansatz = TwoLocal(num_qubits=2, rotation_blocks="ry", entanglement_blocks="cz")
      initial_point = initial_point = [0] * 8
      
      def cost_function(params, ansatz, hamiltonian, estimator):
          energy = estimator.run(ansatz, hamiltonian, parameter_values=params).result().values[0]
          return energy

      # Run VQE using SciPy minimizer routine
      from scipy.optimize import minimize
      result = minimize(cost_function, initial_point, args=(ansatz, hamiltonian, estimator), method="cobyla")

      # Print minimum eigenvalue
      print(result.fun)
```

Reply:
Suggestion added to 6164255417b2ff0f85901759480c047f9e6d08bf Thanks!